core/lexicon/en/resource.inc.php

```diff
@@ -45,6 +45,8 @@
 $_lang['resource_createdon'] = 'Created On';
 $_lang['resource_delete'] = 'Delete';
 $_lang['resource_delete_confirm'] = 'Are you sure you want to delete the resource: "[[+resource]]"?<br />NOTE: Any child resources will also be deleted!';
+$_lang['resource_purge'] = 'Erase resource';
```

Comment:
Here the word "resource" is unnecessary, you can leave only "Erase". Because for the other menu items "resource" is not specified, and the clarification here is unnecessary.

Reply:
oh, yes!, nice catch

Reply:
Or "Permanently erase"?

Reply:
I just took the same text that is on the [trash page](https://github.com/modxcms/revolution/blob/9a3f82cee664c1945aeafd1b232a871c88f7f88a/core/lexicon/en/trash.inc.php#L24)

https://github.com/modxcms/revolution/blob/9a3f82cee664c1945aeafd1b232a871c88f7f88a/core/lexicon/en/trash.inc.php#L24

Reply:
@GulomovCreative I would still remove the clarification, since this is text for the resource tree, and here the clarification looks strange.

![tree_lang](https://github.com/user-attachments/assets/e691b051-682b-4422-bdb5-b1641d150841)